src/styles/_utiles.scss

```diff
@@ -0,0 +1,9 @@
+$color-step: 51;
+$box-hight: 300px;
+$box-minmax-width: minmax(300px, 1fr);
+
+@mixin media-grid($min-width, $rows, $cols) {
+  @media (min-width: $min-width) {
+    grid-template: repeat($rows, $box-hight) / repeat($cols, $box-minmax-width);
+  }
+}
```

Comment:
mixin redundant and complicates the code. I have to scroll for this mixin check what it's doing...

Reply:
Yes, I agree it's redundant now. I needed it before when I didn't use auto-fill